pkg/tailfile/tailfile.go

```diff
@@ -9,6 +9,8 @@ import (
 const blockSize = 1024
 
 var eol = []byte("\n")
+
+// ErrNonPositiveLinesNumber the lines number was negative
 var ErrNonPositiveLinesNumber = errors.New("Lines number must be positive")
```

Comment:
Not related to your changes, but text would be clearer as "The number of lines to extract from the file must be positive"